src/modules/translation/languages/en.json

```diff
@@ -950,13 +951,19 @@
     "add_representation": "Add {bodyShape} representation",
     "change_item_file": "Edit representation"
   },
+  "third_party_collection_detail_page": {
+    "notice": "Each published item takes 1 Slot. {buy_link} to buy Slot Packs and start publishing your creations!",
+    "click_here": "Click here",
+    "slots": "{amount, plural, =0 {No Slots} one {# Slot} other {# Slots}}",
+    "start_adding_items": "Looking good! Start adding items to your new collection",
+    "cant_remove": "You will not be able to add or remove items after publishing your collection."
```

Comment:
The user won't be able to remove already published items, but they will be able to remove unpublished items and add as many items as the slots let them.

Reply:
True! leftover from the old translation